spec/javascript-spec.coffee

```diff
@@ -1138,6 +1220,12 @@ describe "Javascript grammar", ->
       {tokens} = grammar.tokenizeLine('this.obj.prototype = new El()')
       expect(tokens[0]).toEqual value: 'this', scopes: ['source.js', 'variable.language.js']
 
+      {tokens} = grammar.tokenizeLine('$this')
+      expect(tokens[0].value).toEqual '$this'
```

Comment:
Keep them in `expect(tokens[x]).toEqual value: 'value', scopes: ['scopes']` form.